cmd/monaco/download/download_command.go

```diff
@@ -111,7 +111,14 @@ func getDownloadConfigsCommand(fs afero.Fs, command Command, downloadCmd *cobra.
 		},
 		ValidArgsFunction: completion.DownloadDirectCompletion,
 		PreRun: func(cmd *cobra.Command, args []string) {
-			printUploadToSameEnvironmentWarning(args[0], os.Getenv(args[1]))
+			serverVersion, err := client.GetDynatraceVersion2ndGen(client.NewTokenAuthClient(os.Getenv(args[1])), args[0])
```

Comment:
I just realized that we'll need to re-think direct download with OAuth :grimacing:

Reply:
yep